extensions/wikia/AbPerfTesting/AbPerfTesting.setup.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+/**
+ * Performance A/B testing "framework"
+ *
+ * Defines a set of expirements with criterias of when to enable them
+ *
+ * Inspired by https://github.com/etsy/feature
+ *
+ * @see PLATFORM-1246
+ * @author macbre
+ */
+
+$wgExtensionCredits['other'][] = [
+	'name'   => 'A/B Performance Tests',
+	'url'    => 'https://github.com/Wikia/app/tree/dev/extensions/wikia/AbPerfTesting',
+	'author' => [
+		'[http://community.wikia.com/wiki/User:Macbre Maciej Brencz]'
+	],
+];
+
+// generic classes
+$wgAutoloadClasses[ 'Wikia\\AbPerfTesting\\Hooks'                     ] = __DIR__ . '/classes/Hooks.class.php';
```

Comment:
I'd prefer having it namedl "AbPerformanceTests" instead of shortcut-ish "AbPerfTesting" (in all places, not only the namespace but also the master global variable etc.). I'm still open for discussion here.